composer.json

```diff
@@ -6,12 +6,12 @@
   "description" : "XP Compiler",
   "keywords": ["module", "xp"],
   "require" : {
-    "xp-framework/core": "^11.0 | ^10.0",
```

Comment:
We are causing a BC break here by dropping XP 10 support. We have the following options:

* [ ] Backport https://github.com/xp-framework/core/pull/324 to XP 10
* [ ] Create a major version for XP Compiler (9.0.0 at the time of writing)

Reply:
The last XP 10 release, 10.15.1, was released 2021-12-24, more than 1.5 years ago. This should be enough time to upgrade, especially with us looking towards XP 12. On the other side, XP 11 removed tons of deprecated features, which might be a hindrance...

Reply:
Created https://github.com/xp-framework/core/releases/tag/v10.16.0